plugins/modules/inventory_intent.py

```diff
@@ -1127,14 +1163,21 @@ def resync_devices(self):
         device_ips = self.config[0].get("ip_address", [])
 
         if not device_ips:
-            msg = "Cannot perform the Resync operation as device's are not present in Cisco DNA Center"
-            self.status = "failed"
-            self.msg = msg
-            self.log(msg)
+            self.msg = "Cannot perform the Resync operation as device's {0} are not present in Cisco DNA Center".format(str(device_ips))
```

Comment:
`self.msg = f"Cannot perform the Resync operation as device's {device_ips} are not present in Cisco Catalyst Center"
`

Reply:
cannot use f string.